cmd/minishift/cmd/start.go

```diff
@@ -365,7 +365,7 @@ func init() {
 
 // initStartFlags creates the CLI flags which needs to be passed on to 'libmachine'
 func initStartFlags() {
-	startFlagSet.String(isoURL, constants.DefaultIsoUrl, "Location of the minishift ISO.")
+	startFlagSet.String(config.SetUrl.Name, constants.DefaultIsoUrl, "Location of the minishift ISO.")
```

Comment:
Is this why we do:
```diff
+type Flags struct {
+	Name        string
```

if so, this is so meaningless (provides no clue what the relation is, and this also due to the usage `Name` when it is about an URL)